NSS-Day-Cohort-66/rare-api-extra-medium
# Nm authtest

# Created test for authentication and register users

## How to test

1. Pull down NM-authtest
2. Run python3 manage.py test tests -v 1
3. Ensure all test return 'OK'
